diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/FileSize.kt

```diff
@@ -16,7 +16,7 @@ class FileSize : Rule("file-size") {
         private val log = LoggerFactory.getLogger(FileSize::class.java)
         const val MAX_SIZE = 2000L
         private val IGNORE_FOLDER = emptyList<String>()
-        const val IGNORE_FOLDERS_SEPARATOR = ","
+        const val IGNORE_FOLDERS_SEPARATOR = "[,/]"
```

Comment:
Why do you need `/` here? If it is a folder separator, then path will be split by `splitPathToDirs` later.